builtin/providers/aws/resource_aws_route53_record_test.go

```diff
@@ -104,6 +104,9 @@ func testAccCheckRoute53RecordExists(n string) resource.TestCheckFunc {
 }
 
 const testAccRoute53RecordConfig = `
+provider "aws" {
+    region = "us-east-1"
+}
```

Comment:
The Record acceptance tests throw error(s) without this entry in the config now. 
It's related I think to a default config entry that the setup isn't finding for `awslabs/aws-sdk-go`, but I'm not sure where. 

```
2015/02/12 16:26:09 [DEBUG] aws_route53_zone.main: expanding to count = 1
2015/02/12 16:26:09 [INFO] Module root walking: aws_route53_zone.main (Graph node: aws_route53_zone.main)
2015/02/12 16:26:09 [DEBUG] aws_route53_zone.main: Executing Apply
2015/02/12 16:26:09 [DEBUG] Creating Route53 hosted zone: %!s(aws.StringValue=0x1191ca28)
2015/02/12 16:26:10 [ERROR] Error walking 'aws_route53_zone.main': 1 error(s) occurred:

* Credential should be scoped to a valid region, not 'us-west-2'.
```